app/controllers/hyrax/admin/workflows_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Hyrax
+  # Presents a list of works in workflow
+  class Admin::WorkflowsController < ApplicationController
+    before_action :ensure_admin!
+    layout 'admin'
+
+    def index
+      add_breadcrumb t(:'hyrax.controls.home'), root_path
+      add_breadcrumb t(:'hyrax.toolbar.admin.menu'), hyrax.admin_path
+      add_breadcrumb t(:'hyrax.admin.sidebar.tasks'), '#'
+      add_breadcrumb t(:'hyrax.admin.sidebar.workflow_review'), request.path
+      @status_list = Hyrax::Workflow::StatusListService.new(self, "-workflow_state_name_ssim:#{deposited_workflow_state_name}")
```

Comment:
Should we put `workflow_state_name_ssim` behind a new private method that uses Solrizer to construct the key name, or meh?